src/GraphQL.EntityFramework/GraphQL.EntityFramework.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="GraphQL" Version="2.4.0" />
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.2.3" />
```

Comment:
can we do this without the reference?

Reply:
Good point, but unfortunately not. A database provider is required for ConventionSet.CreateConventionSet(context). 

We could move the builder object as an input parameter, so that the package remains database agnostic.